src/components/forms/Radio.jsx

```diff
@@ -11,7 +11,19 @@ import RadioFilledIcon from 'lib/icons/react/RadioFilledIcon';
  * @returns {ReactElement}
  */
 const Radio = React.forwardRef(
-  ({ showLabel = true, disabled = false, name, label, ...rest }, ref) => {
+  (
+    {
+      showLabel = true,
+      disabled = false,
+      name,
+      label,
+      checked,
+      defaultChecked,
+      value,
+      onChange,
+    },
+    ref
```

Comment:
You'll still want to allow passing rest props here just in case the developer wants to add any thing arbitrary like data attributes, event listeners, whatever (though unlike the current example, could you make sure that `className` comes after the rest props? I want to disable anyone from passing `className` through rest props).